src/com/phronemophobic/membrane/term.clj

```diff
@@ -336,21 +339,26 @@
 
 (def default-common-opts {:width 90
                           :height 30
-                          :font-family "monospace"
+                          :font-family :monospace
                           :font-size 12
+                          :toolkit nil
                           :color-scheme default-color-scheme})
 
 (defn run-term
   ([]
    (run-term {}))
   ([opts]
    (let [opts (merge default-common-opts opts)
-         {:keys [width height color-scheme font-family font-size]} opts
+         {:keys [width height color-scheme font-family font-size toolkit]} opts
```

Comment:
It's not clear from reading the code what a toolkit is and what might be valid values.
Perhaps `run-term` and `screenshot` need docstrings.

Reply:
totally agree, https://github.com/phronmophobic/membrane.term/issues/28

Reply:
Ah, good!